src/scene-entry-manager.js

```diff
@@ -428,7 +428,7 @@ export default class SceneEntryManager {
     this.scene.addEventListener("action_selected_media_result_entry", e => {
       // TODO spawn in space when no rights
       const entry = e.detail;
-      if (entry.type === "scene_listing") return;
+      if (entry.type === "scene_listing" && this.hubChannel.permissions.update_hub) return;
```

Comment:
this logic here will make it so users who do not have permission to change the scene will paste links to scenes into the room when using the scene browser, vs having nothing happen at all, which is what happens now.